CONTRIBUTING.md

```diff
@@ -24,21 +24,21 @@ You'll need git and a recent version of Node.JS (any v7.2.1+ is recommended with
 
 **Windows:**
 
-* `npm install --global --production windows-build-tools`
+- `npm install --global --production windows-build-tools`
 
 **OS X:**
 
-* Python
-* Xcode and the Command Line Tools (Xcode -> Preferences -> Downloads), which will install `gcc` and the related toolchain containing `make`.
+- Python
+- Xcode and the Command Line Tools (Xcode -> Preferences -> Downloads), which will install `gcc` and the related toolchain containing `make`.
 
 **Linux:**
 
-* Python v2.7
-* make
-* A proper C/C++11 compiler tool chain, for example GCC
-* Library dependencies:
-  * On Debian-based Linux: `sudo apt-get install libx11-dev libxkbfile-dev execstack libgconf2-4 libsecret-1-dev`
-  * On Red Hat-based Linux: `sudo yum install libX11-devel.x86_64 libxkbfile-devel.x86_64 libsecret-1-dev libsecret-devel`.
+- Python v2.7
+- make
+- A proper C/C++11 compiler tool chain, for example GCC
+- Library dependencies:
+  - On Debian-based Linux: `sudo apt-get install libx11-dev libxkbfile-dev execstack libgconf-2-4 libsecret-1-dev`
```

Comment:
Ahh good catch - thanks for fixing this!

Reply:
had to install it myself on ubuntu, that's how i came across it ;)